cassiopeia/preprocess/utilities.py

```diff
@@ -18,36 +18,77 @@
 import re
 from tqdm.auto import tqdm
 
-from cassiopeia.mixins import is_ambiguous_state
+from cassiopeia.mixins import is_ambiguous_state, logger
 
 
-def generate_log_output(df: pd.DataFrame, begin: bool = False):
-    """A function for the logging of the number of filtered elements.
+def log_moleculetable(wrapped: Callable):
```

Comment:
Since this logs the result of a function (as I understand it), can we add a line to the logger indicating that these are the resulting, not the initial, statistics?